src/Raven.Studio/typescript/viewmodels/shell.ts

```diff
@@ -249,6 +249,7 @@ class shell extends viewModelBase {
                         
```

Comment:
call this.accessManager.secureServer(true); here.

Reply:
sure - can do that - although it is already initialized with true in `accessManager.ts` when declared